### PR TITLE
Add contributing guide for this repo

### PR DESCRIPTION
**Pull request by melchoyce**
See #67.

### CODE REVIEW

**Review by colinmurphy01 — APPROVED** [review]
Made some suggestions for edits and tweaks. Other than these, LGTM

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# contributing.md
```
[the remark this answers is not in the document]
Comment (melchoyce) [reply contributing.md:1]: Great idea 👍

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+If there isn’t an owner, another member of the DA Team can review and approve your work.
+
+#### Who should merge my PR?
+
+This is an [open question](https://docs.google.com/spreadsheets/d/1J8cAcuE2zgpIFSJx_mKOjNmAcQGeYtQbz2vkTuqGI4U/edit#gid=1340104815&range=A22), but for now, if approved, you or another DA team member can merge your PRs.
+
+### Done
+
+Once your work has been approved, or your PR merged, close your issue and move it into the Done column.
```
[the remark this answers is not in the document]
Comment (melchoyce) [reply contributing.md:57]: I feel like I'd need to refer to an engineer here

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+This is work that has been reviewed and prioritized by the team’s executive sponsor. When looking for a task to pick up, this is the column you’ll want to review. If you find an issue you’d like to work on, assign yourself to it and move it over to the Assigned column.
+
+### Assigned
+
+This column is for work that has been assigned to specific team members, but has not yet started. For example, if you assign yourself to a bunch of issues, you can leave them in this column until you start working on them. This is also where you can see at  a glance who to contact if you have a question about a specific issue.
+
+### In progress
+
+Issues that are currently being worked on. Once you’ve assigned yourself to an issue and started working on it, move the issue to this column.
```
[the remark this answers is not in the document]
Comment (melchoyce) [reply contributing.md:23]: Also think I'd be more comfortable with an engineer contributing here. Maybe we could make a follow-up issue to update this with some engineering process clarifications, once the doc is merged?

[the remark this answers is not in the document]
Comment (colinmurphy01) [reply contributing.md:25]: Fair. that'd be helpful.

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+This is currently an [open question](https://docs.google.com/spreadsheets/d/1J8cAcuE2zgpIFSJx_mKOjNmAcQGeYtQbz2vkTuqGI4U/edit#gid=1340104815&range=A14). We have some basic labels for now:
+
+- Blocked: used if the ticket cannot be moved forward
+- Leadership team need: used if the request came from leadership
+- Urgent: sparingly used, only if an issue needs to be considered today or mid-cycle
+- WIP: used if an issue is not ready to be prioritized
+- Billable: used if an issue is billable
+- Design, content, engineering, acquisition, and product: used to note what roles are needed for an issue
+
```
Comment (colinmurphy01) [inline contributing.md:82]:
```suggestion
- Help Me: used if a ticket is in need of a new owner to take it through the finish line.

```

**Review by colinmurphy01 — APPROVED** [review]
One additional suggestion + the idea of drafting a ticket for clarifying DA Team Repo branch management for an eng to work on.

**Pull request merged**